bin/sync_ocp_latest.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+# Set the path to the file that contains the ":OCPLatest:" attribute.
+target=target/downstream/attributes/attributes.adoc
```

Comment:
Just wondering if the code would be easier to read if you used `attributes=` instead of `target=`

Reply:
yeah, good point. I kept it as `target` to be consistent with the other scripts but maybe that's not really necessary. `attributes` would be a little more descriptive here. plus the fact there is no source variable like the other scripts.